brainiak/utils/fmrisim.py

```diff
@@ -1189,6 +1510,7 @@ def _generate_noise_temporal_phys(timepoints,
     ----------
     one dimensional array, float
         Generates the physiological temporal noise timecourse
+
     """
 
     noise_phys = []  # Preset
```

Comment:
Aha, good find, that was actually what I intended to do but did it wrong. In terms of z scoring, I do it in order to make the combination of noise types more tractable. When they are z scored I know how to mix their relative values in order to fit SFNR and SNR

Reply:
For periodic signal, you might get away with analytically calculating the expected variance with magnitude/sqrt(2) according to this wiki page:
https://en.wikipedia.org/wiki/Root_mean_square
That way you do not need to z-score. But of course there may be issue of aliasing. But I think as long as ratio between the frequency of breathing and the fMRI sampling frequency is not a regular number, this estimate should OK.

Reply:
I am not sure what the current issues are with zscoring: at the moment it doesn't really matter that I do it because already the mean is ~0 and std is ~1 but I could remove it if you think it is necessary

Reply:
It is more for reproducibility concern - that you always have the same expected summary statistics for any given type of noise. (i.e., when a user say I simulated certain noise from certain generative model with certain parameters, the parameters that the user put in indeed governs how the samples drawn). I agree it is not a critical issue. So it is up to you.

Reply:
I think I will leave it as is but I appreciate the concern. Given that the critical output of this is the summed response of a respiratory and heart rate oscillation I think the code does what is necessary